File: src/codes/Users_Manual/controller_config.cs
~using System;
~using AUTD3Sharp;
~using AUTD3Sharp.Timer;
~using AUTD3Sharp.Utils;
~_ =
Controller.Builder([new AUTD3(Point3.Origin)])
    .WithDefaultParallelThreshold(4)
    .WithDefaultTimeout(Duration.FromMillis(20))
    .WithSendInterval(Duration.FromMillis(1))
    .WithReceiveInterval(Duration.FromMillis(1))
    .WithTimerStrategy(AUTD3Sharp.Timer.TimerStrategy.Spin(new SpinSleeper()))
~;

File: src/codes/Users_Manual/controller_config.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~(void)
autd3::ControllerBuilder({autd3::AUTD3(autd3::Point3::origin())})
    .with_default_parallel_threshold(4)
    .with_default_timeout(std::chrono::milliseconds(20))
    .with_send_interval(std::chrono::milliseconds(1))
    .with_receive_interval(std::chrono::milliseconds(1))
    .with_timer_strategy(autd3::controller::timer::TimerStrategy::Spin(
        autd3::controller::timer::SpinSleeper()))
    //~;
    //~return 0; }